pyproject.toml

```diff
@@ -0,0 +1,78 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools_scm[toml]>=3.4"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "scitoken-issuer"
+description = "WIPAC - SciToken Issuer"
+readme = "README.md"
+authors = [{name = "WIPAC Developers", email = "developers@icecube.wisc.edu"}]
+keywords = ["transfer"]
+license = {text = "MIT License"}
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+requires-python = ">=3.13"
```

Comment:
Not that we can't set the bar high, but is the 3.13+ intentional?

Reply:
Yep. It only ever runs in a container, so might as well go big.